claimManagement/build.gradle

```diff
@@ -16,7 +16,51 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
+        debug {
+            debuggable = true
+        }
+        flavorDimensions 'std'
+    }
+    flavorDimensions 'std'
+
+    flavorDimensions 'std'
+    productFlavors.all {
+        buildConfigField "String", "API_BASE_URL", '"http://demo.openimis.org/rest/"'
+        buildConfigField "String", "RAR_PASSWORD", '")(#$1HsD"'
     }
+    productFlavors {
+        demoProd {
+            buildConfigField "String", "API_BASE_URL", '"http://demo.openimis.org/rest/"'
```

Comment:
Change applicationId as org.openimis.imisclaims.demoProd. Do this change for other flavors so we can install all versions on same phone.

Reply:
I added this. You can install all build on a single device, but it's hard to determine which is which (nearly all the builds use the same name and icon).

Reply:
Please look on Policies mobile app graddle: https://github.com/openimis/policies_android_app_java/blob/master/app/build.gradle#L51

We are changing the app name based on the build variant. Can you please make this change also?

Reply:
I updated the build variants app names to "Claims (VARIANT)"